docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -4,6 +4,8 @@ title: "Install Camunda 8 on an EKS cluster"
 description: "Set up the Camunda 8 environment with Helm and an optional Ingress setup on Amazon EKS."
 ---
 
+<!-- (!) Note: please keep this guide aligned with docs/self-managed/setup/deploy/openshift/redhat-openshift.md -->
```

Comment:
in what sense keep it aligned?
Maybe we could point out what people should keep aligned?
Structure, implementation details, ...

Reply:
thank you 🙇, get it better know what the alignment idea is!